mandatory/2-function-creation.js

```diff
@@ -16,7 +18,12 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100) {
+    num = 100;
+  }
```

Comment:
This works, but in general it's not recommended to change the value of parameters